jdaviz/app.py

```diff
@@ -1910,10 +1910,10 @@ def _reparent_subsets(self, old_parent, new_parent=None):
         from astropy.wcs.utils import pixel_to_pixel
 
         if isinstance(old_parent, str):
-            old_parent = self.data_collection(old_parent)
+            old_parent = self.data_collection[old_parent]
```

Comment:
I did wonder about this but the `__call__` worked so I didn't make a fuss. What is the difference?

Reply:
This case just wasn't getting used anywhere before, so we didn't see the error.